man/systemd.network.xml

```diff
@@ -992,6 +992,25 @@
             <para>Specifies the outgoing device to match. The outgoing interface is only available for packets originating from local sockets that are bound to a device.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>SourcePort=</varname></term>
+          <listitem>
+            <para>Specifies the source port range match in fib rules. Defaults to unset.</para>
```

Comment:
forwarding information base (FIB)

To be consistent with the rest of the doc...